package.json

```diff
@@ -208,6 +208,7 @@
     "tinymath": "1.2.1",
     "tslib": "^2.0.0",
     "type-detect": "^4.0.8",
+    "unset-value": "^2.0.1",
```

Comment:
this is not bumping existing dependency, you are adding a new dependency. Can you confirm that's the change we need?

Reply:
Though this ```unset-value``` is a sub-dependency, having a potential impact, so I added here as main dependency. Thoughts ?

Reply:
take a look at [this](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2181/files) on how to add resolution for sub-deps

Reply:
`resolutions` would be a better section to place dependency like this one (unset-value) rather than `dependency` section. [Here](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) is more details about the `resolutions` section in `package.json` file.
Specifically this part as mentioned on the `Yarn` website :  
```A sub-dependency of your project got an important security update and you don’t want to wait for your direct-dependency to issue a minimum version update.```